packages/auth-client/src/index.ts

```diff
@@ -284,6 +284,8 @@ export class AuthClient {
 
     // Open a new window with the IDP provider.
     this._idpWindow = window.open(identityProviderUrl.toString(), 'idpWindow') ?? undefined;
+
+    this._eventHandler;
```

Comment:
Was this intentional?

Reply:
That was what you had changed, so I'm guessing yes?